Makefile

```diff
@@ -1,9 +1,9 @@
-CFLAGS = -Wall -g
+CFLAGS = -Wall -g -lSDL2 -lSDL2main -std=c99 -I/usr/local/include/SDL2/
```

Comment:
if SDL2 is installed system-wide in Travis this shouldn't be neccesary, it's worth checking